client/src/components/Reviews.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import EachReview from './EachReview.jsx';
+
+const Reviews = (props) => {
+  const block = {
+    display: 'block',
+  };
+
+  if (props.rating === 0 || props.rating === 'a') {
```

Comment:
I'm sorry but I'm a little confused as to how this section from line 9-21 differs from the return starting on line 22 and as to what the props.rating information is used for. Could you please explain?

Reply:
I have 2 queries for my db: one for getting rated reviews and one for getting all reviews. so props.rating checks if "all" is clicked, then I want to use my ratings prop. if any other rating is clicked, I want to use ratedReviews state